src/Tizen.Multimedia.Remoting/MediaController/MediaControlMetadata.cs

```diff
@@ -36,7 +35,13 @@ public MediaControlMetadata()
 
         internal MediaControlMetadata(IntPtr handle)
         {
-            Debug.Assert(handle != IntPtr.Zero);
+            // If native framework return null handle,
+            // it means server doesn't set metadata yet and it's not error.
+            // So we need to return empty metadata instance as native framework does.
+            if (handle == IntPtr.Zero)
+            {
+                return;
+            }
 
```

Comment:
Not related this change, but you can delete lines to set "Season", "Episode", "Resolution" from this constructor. 
These properties return values from EncodedXXXX property in their getter methods.

Reply:
Right, I removed those lines and uploaded new commit.
Please check it again.